README.md

```diff
@@ -212,6 +210,34 @@ Set of labels would look like this:
 
 Internal and external complainers can have different upload services.
 
+#### Templating
+
+Templates are based on [`text/template`](https://golang.org/pkg/text/template/).
+The following fields are available:
+
+* `nl` - Newline symbol (`\n`).
+* `config` - Function to get labels for the reporter.
+* `failure` - Failure struct.
+* `stdoutURL` - URL of the stdout stream.
+* `stderrURL` - URL of the stderr stream.
+
+With `config` you can use labels in temlates. For example, the following
+template for the Slack reporter:
+
+```
+Task {{ .failure.Name }} ({{ .failure.ID }}) died | {{ config "mentions" }}{{ .nl }}
+```
+
+With the label `complainer_slack_mentions=@devs` will be evaluated to:
+
+```
+Task foo.bar (bar.foo.123) died | @devs
+```
+
+`Failure` struct fields can be found on godoc.org:
+
+* https://godoc.org/github.com/cloudflare/complainer#Failure
```

Comment:
Wouldn`t it make sense to link the failure struct in the list of available fields for templates?
Or even to move this sentence up to the list?
I think it makes more sense, because once you are looking for the possible fields, you expect this link near by (IMO)